android/src/main/java/cl/json/ShareFiles.java

```diff
@@ -109,7 +107,7 @@ private boolean isLocalFile(Uri uri) {
 
             if (this.intentType == null) {
                 this.intentType = type;
-            } else if (type != null && !this.intentType.equalsIgnoreCase(type) && this.intentType.split("/")[0].equalsIgnoreCase((type.split("/"))[0])) {
```

Comment:
same here, can we keep `type != null` ?

Reply:
and same as above - I suppose if paranoid you can keep it, but static analysis indicates it's a nop